ota_proxy/server_app.py

```diff
@@ -169,12 +170,10 @@ async def _pull_data_and_send(self, url: str, scope, send):
                     # also preserved the raw headers value string
                     extra_headers[OTAFileCacheControl.header.value] = header[1].decode()
                 except KeyError:
-                    await self._respond_with_error(
-                        HTTPStatus.BAD_REQUEST,
-                        f"bad OTA-File-Cache-Control headers: {header[1]}",
-                        send,
+                    logger.debug(
```

Comment:
Is it OK to bypass this exception?

Reply:
Emmm yes, after thing twice I think I should revert this change.